test/Quest.t.sol

```diff
@@ -302,6 +308,92 @@ contract TestQuest is Test, TestUtils, Errors, Events {
         quest.withdrawRemainingTokens();
     }
 
+    /*//////////////////////////////////////////////////////////////
+                        CLAIM REFERRAL FEES
+    //////////////////////////////////////////////////////////////*/
+
+    function test_claimReferralFees() public {
```

Comment:
Can we fuzz this test just to check edges? Preference for fuzzed tests whenever there's accounting in play.

Reply:
I updated this to be `testFuzz_claimReferralFees()` and based off the results when running `forge test`, I don't think it's actually running the fuzz tests

`test_fuzz_claimReferralFees()` doesn't work either -- lets sync up on this

Reply:
The name isn't important, you have to pass in a fuzzable arg:

```
function test_someFuzzableFunction(uint256 fuzzableArg) {
  // ...
  contract.someMutativeFunctionWithArguments(fuzzableArg);
  // ...
}
```

But it doesn't really make sense here because the only input is an address which only enables a binary logic branch: either the address has fees to claim or it does not. The purpose of fuzz testing is mainly to catch cases off the happy path that unexpectedly break the contract's invariants. 

Generally, you'd want to either mock out something the function depends on to return a fuzzed value (rather than enforcing the correct execution path by performing the "normal" operations inline as pre-assertion setup), or accept arguments which can be manipulated by the fuzz test. The only thing that seems fuzzable to me in this specific function is the time, since you're warping around and expecting certain behavior depending on the time.

Reply:
Less concerned with the address more concerned with the accounting around the referral payouts since this is the only test function that's covering those accounting paths.